woof-distro/x86/slackware/s15pup32/_00build.conf

```diff
@@ -0,0 +1,323 @@
+#
+#  persistent configuration options
+#
+#  see also DISTRO_SPECS DISTRO_PET_REPOS DISTRO_COMPAT_REPOS-*
+#
+#  **NOTE**: check the original file every once in a while
+#            settings might be added or removed...
+#
+
+# 2createpackages. binaries are usually already stripped. set to 'no' to speed up process
+STRIP_BINARIES=no
+
+## UnionFS: aufs or overlay
+UNIONFS=aufs
+
+#xorg-autoconf from rizalmart - changes perms on /usr/sbin/xorg-autoconf to enable; default is unset
+XAUTOCONF=yes
+
+## Kernel tarballs repo URL for choosing/downloading kernel
+#KERNEL_REPO_URL=http://distro.ibiblio.org/puppylinux/huge_kernels
+
+## Kernel tarball URL
+## avoid being asked questions about downloading/choosing a kernel
+#KERNEL_TARBALL_URL=http://distro.ibiblio.org/puppylinux/huge_kernels/huge-3.4.94-slacko4G2-i686.tar.bz2
+
+## an array of generically named programs to send to the ADRIVE, FDRIVE, YDRIVE
+## ADRV_INC="abiword gnumeric goffice"
+ADRV_INC=""
+## YDRV_INC=""
+YDRV_INC=""
+## FDRV_INC="" #this one is very experimental and it's recommended to be left unset
+FDRV_INC=""
+
+## Include kernel-kit generated FDRIVE
+## set to yes or no or leave commented to be asked the question at build time
+#KFDRIVE=no
+
+## build devx? yes/no 
+BUILD_DEVX=yes
+# build docx, nlsx? yes/no - any other value = no
+BUILD_DOCX=no
+BUILD_NLSX=yes
+
+## include devx SFS in ISO?
+DEVX_IN_ISO=no
+
+## packages to build from source
+PETBUILD_GTK=2
+PETBUILDS="busybox aaa_pup_c gtkdialog jwm puppy_flat_icons puppy_standard_icons gtk_theme_flat_grey_rounded gtk_theme_gradient_grey gtk_theme_polished_blue xdg-puppy-jwm firewallstatus mtpaint"
+
+## Include the windows puppy installer LICK by Luke Lorimer aka <noryb009>
+LICK_IN_ISO=no
+
+## include Pkg in build (y/n). If commented then asked in 3builddistro
+INCLUDE_PKG=y
+
+## compression method to be used (SFS files)
+SFSCOMP='-comp xz -Xbcj x86 -b 512K'
+#SFSCOMP='-comp xz'
+#SFSCOMP='-comp gzip'
+#SFSCOMP='-noI -noD -noF -noX'
+
+## if "$WOOF_HOSTARCH" = "$WOOF_TARGETARCH"
+## This is usually not needed
+EXTRA_STRIPPING=yes
+
+## -- pTheme -- applies only if ptheme pkg is being used
+##    woof-code/rootfs-packages/ptheme/usr/share/ptheme/globals
+## You can choose a ptheme here if you wish
+## otherwise 3builddistro will ask you to choose one
+PTHEME="ScPup"
+#PTHEME="Dark Touch"
+#PTHEME="Dark Mouse"
+#PTHEME="Bright Touch"
+#PTHEME="Bright Mouse"
+#PTHEME="Dark_Blue"
+#PTHEME="Original Pup"
+
+## -- Default Apps --
+## Not all are implemented in the puppy scripts,
+##   but you can specify a default app if you wish...
+## If you specify a value it will override anything that previously
+##   set that value in the corresponding script...
+## These are the current default*apps (scripts) in /usr/local/bin
+DEFAULTAPPS="
+defaultarchiver=
+defaultaudioeditor=
+defaultaudiomixer=
+defaultaudioplayer=
+defaultbrowser=light
```

Comment:
You can use @lakshayrohila's palemoon petbuild if you want a full-featured but relatively small browser without duplicate effort.

Reply:
@dimkr That palemoon PETBUILDS is only fir x86_64.